changelog/unreleased/issue-2739

```diff
@@ -0,0 +1,5 @@
+Bugfix: Make cat command respect --no-lock
+
+Restic cat would not respect the --no-lock flag, this change fixes that issue.
```

Comment:
", this change fixes that issue" => ". It now does". The reader of the changelog is not looking at the commit log.

Reply:
Fixed.